src/arviz_plots/plot_collection.py

```diff
@@ -935,6 +935,7 @@ def map(
         subset_info=False,
         store_artist=True,
         artist_dims=None,
+        step_hist=False,
```

Comment:
map is a generic function it should not have arguments specific arguments of specific visuals